basin3d/synthesis.py

```diff
@@ -477,7 +477,7 @@ def get_timeseries_data(synthesizer: DataSynthesizer, location_lat_long: bool =
                * **start_date**
            Optional parameters for *MeasurementTimeseriesTVPObservation*:
                * **end_date**
-               * **aggregation_duration** = resolution = DAY  (only DAY is currently supported)
+               * **aggregation_duration** = resolution in (SECOND, MINUTE, HOUR, DAY, MONTH)  Default: DAY
```

Comment:
To implement `default: DAY` which I think we should do, I suggest the following be made around new line 490 (sorry, again GitHub won't let me directly suggest).
- Since temporal_resolution is optional, if it is `None` (presumably by default), then make the `kwargs['aggregation_duration'] = TimeFrequencyEnum.DAY`
- I suggest only permitting `SECOND, MINUTE, HOUR, DAY, MONTH` for `temporal_resolution` by adding a check for these values. If check fails, fail to synthesize.

Otherwise, since the hardcoding of `DAY` in `MeasurementTimeseriesTVPObservation` in basin3d/core/synthesis.py was removed, we would be supporting all temporal frequencies / duration_aggregations and not explicitly handling the `NONE` case.

Reply:
https://github.com/BASIN-3D/basin3d/blob/issue-131-timeseries-hires/basin3d/core/schema/query.py#L101-L102

Reply:
Ah, awesome! Thanks for pointing that out. Any thoughts on restricting the temporal_resolution values?